mod-orders/schemas/item_summary.json

```diff
@@ -15,11 +15,21 @@
     },
     "poLineReceiptStatus": {
       "description": "Receipt status of the PO line",
-      "type": "string"
+      "type": "string",
+      "enum": [
+        "Pending",
+        "Open",
+        "Closed"
+      ]
     },
     "poLinePaymentStatus": {
       "description": "Payment status of the PO line",
-      "type": "string"
+      "type": "string",
```

Comment:
These values are different from those in the po_line schema.
It would be great if paymentStatus was extracted to separate schema and reused like [receipt status](https://github.com/folio-org/acq-models/blob/master/mod-orders-storage/schemas/receipt_status.json)
Places where paymentStatus defined:
https://github.com/folio-org/acq-models/blob/8bd9e7d31279bfeb16ef002fd827e29853072be4/mod-orders-storage/schemas/po_line.json#L140
https://github.com/folio-org/acq-models/blob/8bd9e7d31279bfeb16ef002fd827e29853072be4/mod-orders/schemas/composite_po_line.json#L143

Reply:
Changed the code according to the reviewer's remarks.